pep-0699.rst

```diff
@@ -91,6 +91,12 @@ are commonly used, and any increase in their memory consumption will adversely
 affect Python’s performance.
 
 
+Reference Implementation
+========================
+A reference implementation can be found
+`here <https://github.com/python/cpython/pull/101193>`_.
```

Comment:
```suggestion
========================

A reference implementation can be found in
`python/cpython#101193 <https://github.com/python/cpython/pull/101193>`_.
```

Maybe worth being a little more descriptive and unique on the link target, which is important for a11y as well as helps readers in general, and is an overall best practice.

Reply:
Sure.

OT: Perhaps there's a small bit of irony here - but I didn't know what a11y meant and had to google it 😆 .